Test helpers (#19)
Scout24/toguru-client-nodejs: README.md

In general, we recommend parametrizing your main application to take a toguru client interface, and instantiating the client itself at the very edge of your application (typically `index.ts`). This allows you to use different clients during tests that you can fully control without the need to introduce any mocking tools.

> ### Express

Reviewer: Can you also run the TOC (see top of readme) so it gets update with the new section). Thanks

> If your express application follows the pattern described above your application should approsimatively look like this:

Reviewer: English plus simplify

```suggestion
Following the pattern align above, a typical `Express` application will look like 
```

```ts
const application = (

>   toguruMiddleware: (req: Request, resp: Response, next: NextFunction) => void

Reviewer: We can reference the actual express type, it's shorter

```suggestion
    toguruMiddleware: Express.RequestHandler
```

) => {
  const app = express()
  app.get('/toguru-test', toguruMiddleware,...)
})
```

> In tests we can pass the `stubToguruMiddleware`/`stubClient`, depending on what you chosed to use, that allows you to set the toggles state.

Reviewer: ```suggestion
During tests, we can pass the `express.stubToguruMiddleware` or `express.stubClient`, depending on what the app is using, that easily allows you to instantiate the middleware/client with a given set of toggles, falling back to the toggle defaults for the rest.
```

The appli
